test/fake-ws.js

```diff
@@ -70,19 +66,19 @@ function resetCursor () {
 }
 
 function processQueue () {
-    let f;
+    let currentClientMessage;
```

Comment:
Well, it is not a message but rather a message handler I would say, it is a function.

Reply:
What do you think of renaming "f" to "currentHandler" and the queue itself from "clientMessagesQueue" to "clientMessageHandlersQueue" ?

Reply:
👍🏻  great!